contrib/gnome-tweaks/template.py

```diff
@@ -1,21 +1,40 @@
 pkgname = "gnome-tweaks"
-pkgver = "45.2"
+pkgver = "46.0"
 pkgrel = 0
 build_style = "meson"
-hostmakedepends = ["meson", "pkgconf", "gettext"]
-makedepends = ["libhandy-devel"]
+hostmakedepends = [
+    "desktop-file-utils",
+    "gettext",
+    "gobject-introspection",
+    "gtk-update-icon-cache",
+    "meson",
+    "pkgconf",
+]
+makedepends = [
+    "glib-devel",
+    "gsettings-desktop-schemas-devel",
+    "gtk4-devel",
+    "libadwaita-devel",
+    "libgudev-devel",
+    "python-gobject-devel",
+]
 depends = [
-    "gtk+3",
-    "dconf",
-    "mutter",
+    "glib",
+    "gnome-desktop",
     "gnome-settings-daemon",
+    "gnome-shell",
+    "gsettings-desktop-schemas",
+    "gtk4",
+    "libadwaita",
+    "libgudev",
```

Comment:
also redundant

Reply:
i added both this and glib for explicitness but okay i'll remove them